networkmanager/networkmanager_test.go

```diff
@@ -18,37 +18,65 @@
 package networkmanager_test
 
 import (
+	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net"
 	"os"
-	"os/exec"
 	"path"
-	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/aoscloud/aos_common/aoserrors"
-	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
+	cni "github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/types"
+	current "github.com/containernetworking/cni/pkg/types/100"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/aoscloud/aos_common/aoserrors"
```

Comment:
aos_common should be part of 3rf parties pacakges.

Reply:
Done